corehq/apps/app_manager/tests/test_build_errors.py

```diff
@@ -22,7 +23,7 @@ def _clean_unique_id(errors):
             if 'module' in error and 'unique_id' in error['module']:
                 del error['module']['unique_id']
 
-    def test_subcase_errors(self, mock):
+    def test_subcase_errors(self, mock1, mock2):
```

Comment:
could you avoid all these unused parameters with `*args`? i.e:

```py
def test_subcase_errors(self, *args):
    ...
```

Reply:
ooh, yes, that's much nicer - fdccff37ef7